src/Marketing/InstalledExtensions.php

```diff
@@ -152,6 +158,38 @@ protected static function get_facebook_extension_data() {
 		return $data;
 	}
 
+	/**
+	 * Get Pinterest extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_pinterest_extension_data() {
+		$slug = 'pinterest-for-woocommerce';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = plugins_url( 'images/marketing/pinterest.svg', WC_ADMIN_PLUGIN_FILE );
+
+		// TODO: Finalise docs url.
```

Comment:
Not finalised yet – this may be a while off (~weeks). This is the likely url, so maybe best to proceed with this and we can update later. 

Let me know if you want the `TODO` comment removed, probably not adding much value :)

Reply:
@haszari Sound good, let's keep the `TODO` 😄